backend/server.js

```diff
@@ -40,3 +41,65 @@ app.listen(config.port, config.host, (e) => {
   }
   logger.info(`${config.name} running on ${config.host}:${config.port}`);
 });
+
+//POST -> /auction -> create auction
+app.post('/auction', async (req,res)=>{
+  var id = req.param('id');
+  var current_bid= req.param('current_bid');
+  var list_user_id= req.param('list_user_id');
+  var bid_user_id= req.param('bid_user_id');
+  var product_id= req.param('product_id');
+  var show_user_bid= req.param('show_user_bid');
+  var is_finished= req.param('is_finished');
+  var start_date= req.param('start_date');
+  var end_date= req.param('end_date');
+  var description= req.param('description');
+
+  config.query("INSERT INTO auction (id,current_bid,list_user_id,bid_user_id,product_id,show_user_bid,is_finished,start_date,end_date,description) VALUES (?,?,?,?,?,?,?,?,?,?)",[id,current_bid,list_user_id,bid_user_id,product_id,show_user_bid,is_finished,start_date,end_date,description],function (err, result, fields) {
+    if (err) throw err;
+    res.end(JSON.stringify(result)); // Result in JSON format
```

Comment:
This should be in the form
res.status(200).send({ success: true, msg: rows[0] });